coops-ui/src/views/StandaloneOfficeAddressFiling.vue

```diff
@@ -445,10 +486,33 @@ export default {
       }
 
       if (this.isDataChanged('OTADD') && this.addresses) {
-        changeOfAddress = {
-          changeOfAddress: {
-            deliveryAddress: this.formatAddress(this.addresses['deliveryAddress']),
-            mailingAddress: this.formatAddress(this.addresses['mailingAddress'])
+        if (this.addresses.recordsOffice) {
```

Comment:
I'd prefer to see this logic have "common" code and then conditional code as needed. Eg,
```
myObject['address1'] = { address 1 fields }
if (address2) {
   myObject['address2'] = { address2 fields }
}
```

IMHO this would be easier to understand. (Also, DRY.)

Reply:
Future cleanup maybe.

Reply:
I understand where you're coming from completely. I had condensed with conditionals in the past and it was requested for the code to be expanded and simplified, and with this i took that approach. It could also be assigned after creation with a conditional.